File: lib/utils/signup_complete.dart
import 'package:develove/utils/constants.dart';

Future<void> updateUser(String fullName, List<String> tags, String? username) async {
  var data = {
    'username': username ?? await supabase.auth.currentUser!.email,
    'email': await supabase.auth.currentUser!.email,
    'fullName': fullName,
    'tags': tags,
  };
  final res = await supabase.from('users').insert(data).execute();
  print(res.status);
  print(res.error);
}

File: web/lib/signupComplete.js
import { supabase } from "./supabase";

export async function updateUser(fullName, tags, username = null) {
  let data = {
    username: username != null ? username : supabase.auth.user().email,
    email: supabase.auth.user().email,
    fullName,
    tags,
  };
  const res = await supabase.from("users").insert(data);
  console.log(res.data);
  console.log(res.error);
}
